### PR TITLE
add animation to tech block

### PR DESCRIPTION
**Pull request by joredudiaz**
## Summary

- Tech animation transition

## Jira Story

https://trello.com/c/gBCqzTik

## Screenshot

transition

### CODE REVIEW

File: mock-up/scss/tech.scss [hunk 1]
```diff
@@ -17,6 +17,7 @@
   padding: 10px;
   position: relative;
   width: 225px;
+  transition: after 0.5s ease;
```
Comment (betoobernal) [inline mock-up/scss/tech.scss]: the transition is for the pseudoelement. Carry it to `&::after`

Comment (betoobernal) [inline mock-up/scss/tech.scss]:
```suggestion
  transition: right 0.5s ease, bottom 0.5s ease;
```
They are only for css properties. (not all)

File: mock-up/scss/tech.scss [hunk 2]
```diff
@@ -25,13 +26,20 @@
...
+    right: 10px;
     width: 225px;
     z-index: -1;
   }
 }
 
+.tech:hover {
+  &::after {
+    top: 10px;
+    left: 10px;
```
Comment (betoobernal) [inline mock-up/scss/tech.scss]: The transition is only for `right` and `bottom` properties.
Try this:
```suggestion
    right: -10px;
    bottom: -10px;
```
pd:
this is an exceptional case 😅

File: mock-up/scss/tech.scss [hunk 3]
```diff
@@ -28,12 +28,20 @@
...
   }
 }
 
+.tech:hover {
+  &::after {
+    bottom: -10px;
+    right: -10px;
+  }
+}
+
```
Comment (FrankIglesias) [inline mock-up/scss/tech.scss]:
```suggestion
&:hover::after {
    bottom: -10px;
    right: -10px;
  }
 }

```

Comment (FrankIglesias) [reply mock-up/scss/tech.scss]: Check indentation before applying this change

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by betoobernal — APPROVED** [review]

**Pull request merged**